src/main/java/zmq/socket/pubsub/Dist.java

```diff
@@ -7,7 +7,7 @@
 import zmq.Msg;
 import zmq.pipe.Pipe;
 
-class Dist
+public class Dist
```

Comment:
Is this meant to be public or is protected sufficient?

Reply:
I used the same modifier LB and FQ is using for consistency